starport/cmd/account.go

```diff
@@ -0,0 +1,99 @@
+package starportcmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+	"github.com/tendermint/starport/starport/pkg/cosmosaccount"
+)
+
+const (
+	flagAddressPrefix = "address-prefix"
+)
+
+const (
+	flagPassphrase     = "passphrase"
+	flagNonInteractive = "noninteractive"
```

Comment:
```suggestion
	flagNonInteractive = "non-interactive"
```

Reply:
Is it like `force` ?

Reply:
no, this is a security feature.

when it's used, interactive mode for entering passphrase will be disabled (insecure) and you need to set the --passphrase instead if you have one.

Reply:
Ok I see. Can't we use this mode by default if `--passphrase` is used?

Otherwise, I still think it's cleaner to have `non-interactive` instead of `noninteractive`

Reply:
> Ok I see. Can't we use this mode by default if --passphrase is used?

this is already the case.

Reply:
Why shouldn't we use `non-interactive` instead of `noninteractive`?

Reply:
please check the commits I already applied your all suggestions